File: dongwoo/week1/easy.java
import java.util.*;
class Solution {
    public int solution(int[] nums) {
        int answer = 0;
        int max = nums.length / 2;
        Set<Integer> set = new HashSet<>();
        for (int num : nums) {
            set.add(num);
        }

        if (set.size() >= max) {
            return max;
        }

        return set.size();
    }
}

File: jugwang/week4/easy.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> array) {
    int answer = 0;
    
    vector<int>::iterator ptr;
    for (ptr = array.begin(); ptr != array.end(); ++ptr) {
        int num = *ptr;
        while(num != 0) {
            if (num %10 == 7) {
                answer++;
            }
            num /= 10;
        }
    }

    return answer;
}